forum/qa-plugin/q2a-badges-master/qa-badge-check.php

```diff
@@ -358,20 +358,17 @@ function answer_vote_down($event,$event_user,$params) {
 			if($event_user) $this->check_voter($event_user);
 		}
 
-		function check_voted($uid) {
-						// upvotes received
+                public function check_voted($uid) {
+                    // upvotes received
 			
-			$votes = qa_db_read_one_assoc(
-				qa_db_query_sub(
-					'SELECT upvoteds FROM ^userpoints WHERE userid=#',
-					$uid
-				),
-				true
-			);
-			$badges = array('liked','loved','revered');
+                    $votesArray = qa_db_query_sub('SELECT upvoteds FROM ^userpoints WHERE userid=# LIMIT 1', $uid);

```

Comment:
usunąłeś flagę true, która mówi o tym że qa_db_query_sub() może dopuszczać puste wyniki - czy to celowe?